src/shared/utils/http-utils.js

```diff
@@ -0,0 +1,75 @@
+import $ from 'jquery';
```

Comment:
jQuery is evil. Don't use it. https://discuss.reactjs.org/t/jquery-with-react/683
node-request can be a replacement here.